tsconfig.base.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "lib": [
+      "es2017"
+    ],
+    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    "target": "es2017",
+    "strict": true,
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
+  },
+  "include": [
```

Comment:
You don't need to have the `include` here.

Check out https://github.com/nteract/outputs for an example of how this should be structured.